src/algorithms/klminwassfwdbwd.jl

```diff
@@ -41,60 +41,6 @@ The keyword arguments are as follows:
     subsampling::Sub = nothing
 end
 
-"""
```

Comment:
This portion has been moved to a separate file `algorithms/gauss_expected_grad_hess.jl`.